tox.ini

```diff
@@ -59,9 +59,9 @@ deps =
     numpy124: numpy==1.24.*
     numpy125: numpy==1.25.*
 
-    mpl334: matplotlib==3.3.4
-    # mpl 3.3.4 isn't compatible with numpy 2.0 but didn't include a pin
-    mpl334: numpy<2.0
+    mpl350: matplotlib==3.5.0
+    # mpl 3.5.0 isn't compatible with numpy 2.0 but didn't include a pin
```

Comment:
> mpl 3.5.0 isn't compatible with numpy 2.0 but didn't include a pin

Is this actually true?

Reply:
Okay, maybe still true according to https://github.com/numpy/numpy/issues/26191 . Nvm

Reply:
I just checked. The pin only appeared in mpl 3.8.0